daktari/checks/terraform.py

```diff
@@ -7,26 +7,40 @@
 from daktari.os import OS
 
 
+class TfenvInstalled(Check):
+    name = "tfenv.installed"
+
+    suggestions = {
+        OS.OS_X: "<cmd>brew install tfenv</cmd>",
+        OS.UBUNTU: """<cmd>git clone https://github.com/tfutils/tfenv.git ~/.tfenv</cmd>
+                    <cmd>ln -s ~/.tfenv/bin/* /usr/local/bin</cmd>""",
```

Comment:
Do we need a couple extra spaces here for it to line up?

Reply:
I tested it locally and it looked fine, but I can align it for code neatness if nothing else :grin:

Reply:
![image](https://user-images.githubusercontent.com/57534485/125105896-e4222c00-e0d6-11eb-9b7c-b13495eda860.png)

Reply:
Maybe it trims initial spaces, I forget now!